draft-ietf-tls-dtls13.md

```diff
@@ -1917,8 +1917,10 @@ NewConnectionId with usage "cid_spare" containing num_cid CIDs soon as
 possible.  Endpoints MUST NOT send a RequestConnectionId message
 when an existing request is still unfulfilled; this implies that
 endpoints needs to request new CIDs well in advance.  An endpoint MAY
-ignore requests, which it considers excessive (though they MUST be
-acknowledged as usual).
+handle requests which it considers excessive by responding with
+a NewConnectionId message containing fewer than num_cid CIDs,
+including no CIDs at all. Endpoints MAY handle an excessive number
+of RequestConnectionId messages by terminating the connection.
```

Comment:
Hmmm. This sounds like a fragile solution to me unless we introduce a new error here to give the peer some idea why the connection was closed.

Reply:
Well, we could add a new alert: "excessive_rcid_requests"

Reply:
@chris-wood WDYT?

Reply:
Alerts are cheap, so I'd be fine with that.

Reply:
@ekr can you update this PR with a new dedicated alert?